AcadeMeData/tests.py

```diff
@@ -43,4 +43,48 @@ def test_get_user(self):
         # users_list = User.objects.all()
         # assert users_list[0].user.username == "user5"  # the first user in 0002_User_test_data
         assert User.get_user('username') == user_for_example.user
-        # assert users_list[len(users_list) - 1] == user_for_example
+
+
+@pytest.mark.django_db
+class TestUniversityModel:
+    def university_example(self):
+        university = University(university_id=1, name='Reichman University', location="Herzlia",
+                                description="A nice place")
+        university.save()
+        return university
+
+    def test_get_university_by_name(self):
+        test_university = self.university_example()
+        university_test = University.get_university_by_name(
+            'Reichman University')
+        assert test_university == university_test
+        assert isinstance(university_test, University)
+
+    def test_get_university_by_location(self):
+        test_university = self.university_example()
+        university_test = University.get_university_by_location(
+            'Herzlia')
+        assert test_university == university_test
+        assert isinstance(university_test, University)
+
+
+@pytest.mark.django_db
+class TestProffesorModel:
+    def proffesor_example(self):

```

Comment:
1. I would change the function name to get_proffesor() to imply what it does
2. There is an opportunity to make it a more generic function with the following signature

get_professor(professor_id=1, name="...")

Overall, try to avoid having fixed strings in your functions and methods.

Reply:
Changed, RESOLVED